src/test/regress/sql/multi_insert_select_window.sql

```diff
@@ -2,9 +2,7 @@
 -- test insert select functionality for window functions
 -- ===================================================================
 
-TRUNCATE agg_results;
-
-INSERT INTO agg_results (user_id, agg_time, value_2_agg)
+INSERT INTO agg_results_window (user_id, agg_time, value_2_agg)
```

Comment:
We could consider using different schema and search paths for different tests.

Reply:
This was the general attitude on multi_behavioral_analytics tests, so I just continued but I can apply what you suggest to all of the tests that I run parallel?

Reply:
Makes sense, let's consider it for the next changes. But, this change should be OK for now?

Reply:
It probably makes sense, that way you can also just drop the schema at the end of the test and that would even speed up things a bit further (due to DROP TABLE in separate transactions being slow).

Reply:
Yes, we can merge this.